doc/appendix-de-kassensichv/reference-tables/reference-tables.md

```diff
@@ -279,3 +279,474 @@ For definitions regarding national laws, please refer to the appropriate appendi
+| `Z_ENDE_ID` | Last BON_ID in the closing | String (40) | Automatically filled by ft |
+| `NAME` | Name of the company | String (60) | Mandatory, `dailyClosingMasterData.CompanyName` |
+| `STRASSE` | Street | String (60) | Mandatory,`dailyClosingMasterData.CompanyStreet` |
+| `PLZ` | Zip | String (10) | Mandatory, `dailyClosingMasterData.CompanyZip` |
+| `ORT` | City | String (62) | Mandatory, `dailyClosingMasterData.CompanyCity` |
+| `LAND` | Country | ISO 3166 ALPHA-3 country code | Mandatory, `dailyClosingMasterData.CompanyCountry` |
+| `STNR` | Tax number of the company | String (20) | Mandatory, `dailyClosingMasterData.CompanyTaxNr` |
+| `USTID` | VAT ID | String (15) | Mandatory, `dailyClosingMasterData.CompanyVATId` |
+| `Z_SE_ZAHLUNGEN` | Total of all payments | Decimal (2) | Automatically filled by ft  |
+| `Z_SE_BARZAHLUNGEN` | Total of all cash payments | Decimal (2) | Automatically filled by ft |
+
+(`dailyClosingMasterData`is described above in the chapter "Master data module")
+
+##### File: Stamm_Orte (location.csv)
+
+| **Fieldname**            | **Description**          | **Format**          | **ft.input** |
+|----------------------|--------------------------|---------------------|---------------------|
+| `Z_KASSE_ID` | ID of the (closing) cashpoint | String (50) | `ftCashBoxIdentification` |
+| `Z_ERSTELLUNG` | Date of the cashpoint closing | ISO 8601 und RFC3339 date | from `cbReceiptMoment` of the daily closing receipt |
+| `Z_NR` | Nr. of the cashpoint closing | Integer | Automatically filled by ft |
+| `LOC_NAME` | Name of the site | String (60) | Mandatory, `dailyClosingMasterData.LocationName` |
+| `LOC_STRASSE` | Street | String (60) | Mandatory, `dailyClosingMasterData.LocationStreet` |
+| `LOC_PLZ` | Zip | String (10) | Mandatory, `dailyClosingMasterData.LocationZip` |
+| `LOC_ORT` | City | String (62) | Mandatory, `dailyClosingMasterData.LocationCity` |
+| `LOC_LAND` | Country | ISO 3166 ALPHA-3 county code | Mandatory, `dailyClosingMasterData.LocationCountry` |
+| `LOC_USTID` | VAT ID | String (15) | Mandatory, `dailyClosingMasterData.LocationVATId` |
+
+(`dailyClosingMasterData`is described above in the chapter "Master data module")
+
+##### File: Stamm_Kassen (cashregister.csv)
+
+| **Fieldname**            | **Description**          | **Format**          | **ft.input** |
+|----------------------|--------------------------|---------------------|---------------------|
+| `Z_KASSE_ID` | ID of the (closing) cashpoint | String (50) | `ftCashBoxIdentification` |
+| `Z_ERSTELLUNG` | Date of the cashpoint closing | ISO 8601 und RFC3339 date | From `cbReceiptMoment` of the daily closing receipt |
+| `Z_NR` | Nr. of the cashpoint closing | Integer | Automatically filled by ft |
+| `KASSE_BRAND` | Brand of the cash register | String (50) | Mandatory, `dailyClosingMasterData.CashRegisterBrand` |
+| `KASSE_MODELL` | Model designation | String (50) | Mandatory, `dailyClosingMasterData.CashRegisterModel` |
```

Comment:
From the https://portal-sandbox.fiskaltrust.de/POSSystem - PosSystemID: KASSE_MODELL, KASSE_SW_BRAND, KASSE_SW_VERSION, KASSE_BASISWAEH_CODE (actualy not existing)
KEINE_UST_ZUORDNUNG - already discussed.

KASSE_SERIENNR =? ftCashBoxID ?